### DISCUSSION TITLE
quarkus-resteasy-reactive-jackson with JsonView AND SecureField

### DISCUSSION THREAD

**Discussion by lenped** [quarkusio/quarkus#28901] [Q&A]
In the Quarkus guide https://quarkus.io/guides/resteasy-reactive#json-serialisation  JsonView and SecureField is mentioned as supported. This seems to work if I am using one OR the other, but if I try to use both in same entity only securefield will work. Is this a bug?

[a comment by an automated account is not shown]

**Comment by geoand** [comment]
It does sound like a bug. Can you please open an issue and attach a sample project showing the problem if possible?

Thanks

**Comment by sberyozkin** [comment]
Hey all, created #29010 to track it

**Comment by gsmet** [comment]
This has been solved here: https://github.com/quarkusio/quarkus/issues/29010 .
